packages/snaps-execution-environments/lavamoat/build-system/policy-override.json

```diff
@@ -1,8 +1,119 @@
 {
   "resources": {
-    "lavamoat-browserify": {
+    "@swc/core": {
+      "packages": {
+        "@swc/core>@swc/core-darwin-arm64": true,
+        "@swc/core>@swc/core-linux-x64-gnu": true
```

Comment:
These are platform-specific, so the LavaMoat policy would change depending on the platform it's built on. I've only added macOS and Linux for now, but we should add Windows here as well.

`@swc/core` is only used for the terser as far as I can tell, so potentially we can remove it completely.

Reply:
How come we're not adding windows now? I'm confused, why would we remove @swc/core if it's used for the terser?

Reply:
> How come we're not adding windows now?

I don't have a Windows PC so I'm not really sure what to add 😅

> I'm confused, why would we remove @swc/core if it's used for the terser?

It's used by Webpack itself. We don't have a direct dependency on `@swc/core` anymore.

Reply:
As far as I can tell we don't actually use `@swc/core` anymore? It also runs on Windows without any additions

Reply:
It's still included in the build policy if I remove the overrides 😕

Reply:
f5c19bc60b88688a4ce526aca06b517435327f27

Reply:
It doesn't seem to be actually used at runtime. Does it not run on Mac OS without the override?

Reply:
It works without the override because it's added to the build policy. I added the override because the generated policy would be different on different platforms.

Reply:
Well, if you let the CI update it to just be the Linux policy, does it not work on your end?